app/src/main/res/raw/SampleData.csv

```diff
@@ -0,0 +1,7 @@
+﻿Key,Name,Latitude,Longitude,Street Address,City,State,Zip,Type,Phone,Website
```

Comment:
res/raw files can't have uppercase apparently (it wouldn't build) but renaming to `sampledata.csv` worked for me